.github/workflows/main.yml

```diff
@@ -22,109 +22,109 @@ jobs:
         # https://hexdocs.pm/elixir/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp
         include:
           # Newest supported Elixir/Erlang pair.
-          - elixir: '1.18'
-            otp: '27.2'
+          - elixir: "1.18"
+            otp: "27.2"
             lint: true
             dialyzer: true
             os: ubuntu-latest
 
           # One version before the last supported one.
-          - elixir: '1.17'
-            otp: '26.2'
+          - elixir: "1.17"
+            otp: "26.2"
             os: ubuntu-22.04
 
           # Oldest supported Elixir/Erlang pair.
-          - elixir: '1.13.4-otp-22'
-            otp: '22.3.4'
-            os: ubuntu-20.04
+          - elixir: "1.13"
+            otp: "25.3"
+            os: ubuntu-22.04
 
     steps:
-    - name: Check out this repository
-      uses: actions/checkout@v4
-
-    - name: Setup Elixir and Erlang
-      uses: erlef/setup-beam@v1
-      with:
-        elixir-version: ${{ matrix.elixir }}
-        otp-version: ${{ matrix.otp }}
-
-    - name: Cache downloaded dependencies
-      uses: actions/cache@v4
-      id: mix-downloaded-deps-cache
-      with:
-        path: |
-          deps
-          test_integrations/phoenix_app/deps
-        key: |
-          ${{ runner.os }}-${{ matrix.otp }}-${{ matrix.elixir }}-mix-deps-${{ hashFiles('**/mix.lock') }}
-
-    - name: Download Mix dependencies
-      if: steps.mix-downloaded-deps-cache.outputs.cache-hit != 'true'
-      run: |
-        if [ ${{ matrix.elixir }} != '1.13.4-otp-22' ]; then
-          mix deps.get --check-locked
-        else
-          mix deps.get
-        fi
-
-        if [ ${{ matrix.elixir }} != '1.13.4-otp-22' ]; then
-          cd test_integrations/phoenix_app
-
-          if [ ${{ matrix.lint }} == 'true' ]; then
+      - name: Check out this repository
+        uses: actions/checkout@v4
```

Comment:
The indentation here is not a necessary change and makes the diff harder to review ☹️ Maybe we can split it up into a separate PR, because it's going to pollute the blame if someone needs to debug this...